platform/files/report.go

```diff
@@ -0,0 +1,21 @@
+package files
+
+import "github.com/buildpacks/lifecycle/buildpack"
+
+// Report is written by the exporter as report.toml to record information about the build.
+// It is not included in the output image, but can be saved off by the platform before the build container exits.
```

Comment:
in general these comments are great and answer the primary questions I would have (what file name matches this struct, what phase writes it, how can i find it, etc).   In this case (and maybe any similar ones i skimmed past) would it be reasonable to name the flag that causes it to be output?   e.g. `but will be saved off by the platform before the build container exits if the --report flag is provided" ?

Reply:
Sounds good! Updated in 09c47792